### PR TITLE
tetragon: Allow multiple symbol instances in kprobe spec

### PR DESCRIPTION
**Pull request by olsajiri**
it's ok for non kprobe-multi specs and we retrn error in kprobe-multi mode

Fixes: https://github.com/cilium/tetragon/issues/2947

### CODE REVIEW

File: pkg/sensors/tracing/generickprobe.go [hunk 1]
```diff
@@ -897,11 +912,16 @@ func createKprobeSensorFromEntry(kprobeEntry *genericKprobe,
 	loadProgName, loadProgRetName := kernels.GenericKprobeObjs()
 	isSecurityFunc := strings.HasPrefix(kprobeEntry.funcName, "security_")
 
+	pinProg := kprobeEntry.funcName
+	if kprobeEntry.instance != 0 {
+		pinProg = fmt.Sprintf("%s_%d", kprobeEntry.funcName, kprobeEntry.instance)
```
Comment (kkourt) [inline pkg/sensors/tracing/generickprobe.go:917]: How about we use "%s:%d"? I'm just thinking that in theory there could be a fund name "foo_1" in the kernel that may cause collisions.

File: pkg/sensors/tracing/generickprobe.go [hunk 2]
```diff
@@ -986,6 +1006,9 @@ func createKprobeSensorFromEntry(kprobeEntry *genericKprobe,
 
 	if kprobeEntry.loadArgs.retprobe {
 		pinRetProg := sensors.PathJoin(fmt.Sprintf("%s_return", kprobeEntry.funcName))
+		if kprobeEntry.instance != 0 {
+			pinRetProg = sensors.PathJoin(fmt.Sprintf("%s_%d_return", kprobeEntry.funcName, kprobeEntry.instance))
```
Comment (kkourt) [inline pkg/sensors/tracing/generickprobe.go:1010]: ditto

**Review by kkourt — APPROVED** [review]
Left a minor comment, but otherwise LGTM.
Thanks!

File: pkg/sensors/tracing/generickprobe.go (hunk 1 above)
Comment (olsajiri) [reply pkg/sensors/tracing/generickprobe.go:917]: ook, will change, thanks

**Pull request merged**